proxmox/config_qemu.go

```diff
@@ -126,6 +127,11 @@ func (config ConfigQemu) CreateVm(vmr *VmRef, client *Client) (err error) {
 	// Create disks config.
 	config.CreateQemuDisksParams(vmr.vmId, params, false)
 
+	// Create vga config.
+	vgaParam := QemuDeviceParam{}
+	vgaParam = vgaParam.createDeviceParam(config.QemuVga, nil)
+	params["vga"] = strings.Join(vgaParam, ",")
```

Comment:
if this is empty, can we omit the params so it acts just like before?

Reply:
Modified.
When I create a VM without the parameter it sets serial0 as Display but the documentation indicates std as default parameter.